src/website/app/VariableTableTitle.js

```diff
@@ -0,0 +1,43 @@
+/* @flow */
```

Comment:
Just confirmed that these styles, save for the `textTransform: 'capitalize'` are exactly the same as those used for ComponentDocExample's title. It'd be great to rename this to something more generic (maybe SiteSubHeading?) and use in ComponentDocExample, Theming/index, and Tokens/index.